server/components/fill_teams_test.js

```diff
@@ -0,0 +1,46 @@
+/**
+* Created by Sammy on 1/13/15
+**/
+
+var db = require('../api/models');
+
+var Fill = (function() {
+  var stockTeams = [
+    {name: "Albany Alphas"},
+    {name: "Alaska Arctics"},
+    {name: "Baltimore Spirits"}
+  ]
+
+  var stockPlayers = [
+    {name: "Rodger Umaechi",
+    realWorldTeam: "ALB"},
+    {name: "Apiatan Redmane",
+    realWorldTeam: "ALB"},
+    {name: "Carl Esteban",
+    realWorldTeam: "ALK"},
+    {name: "Ambrose Slade",
+    realWorldTeam: "ALK"},
+    {name: "Silvia Windcreek",
+    realWorldTeam: "BAL"},
+    {name: "Dirk von Stryker",
+    realWorldTeam: "BAL"}
+  ]
+
+  var _fillTeams = function(user) {
+    if (user.role !== 'admin') {
+      console.log("Error, user is not an admin, user role is: " + user.role);
+      return false;
+    }
+    return db.Team.bulkCreate(stockTeams)
+  }.then(function() {
```

Comment:
If (user.role !== 'admin') you end up calling false.then() which isn't a function. If you want to  keep chaining promises off of fillTeams in this case, you would return Promise.resolve(false) and then the next function in the promise chain would get false as its argument. If you want the promise chain to quit, you can use Promise.reject(XXX) where XXX is the rejection reason -- then if you ever handle a promise rejection case in the chain, you'll get XXX as the parameter.